PowerShell.Common.props

```diff
@@ -96,8 +96,8 @@
     <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
 
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <RuntimeFrameworkVersion>3.0.0-preview-27122-01</RuntimeFrameworkVersion>
-    <LangVersion>Latest</LangVersion>
+    <RuntimeFrameworkVersion>3.0.0-preview-27324-05</RuntimeFrameworkVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Why do we set explicit value If latest should be 8.o now?

Reply:
The build failed complaining some language feature was only available in 8.0, making this explicit fixed it.

Reply:
I wonder - we still do not use 8.0 features.

Reply:
@iSazonov true, did a search around the netcore repos and didn't find anything related to this.  This is the error if you leave it as `latest`:

```output
/Users/steve/repos/PowerShell/src/System.Management.Automation/engine/COM/ComUtil.cs(318,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/Users/steve/repos/PowerShell/src/System.Management.Automation/security/Authenticode.cs(193,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/Users/steve/repos/PowerShell/src/System.Management.Automation/security/Authenticode.cs(516,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/Users/steve/repos/PowerShell/src/System.Management.Automation/security/Authenticode.cs(541,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/Users/steve/repos/PowerShell/src/System.Management.Automation/security/Authenticode.cs(611,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/Users/steve/repos/PowerShell/src/System.Management.Automation/security/nativeMethods.cs(1070,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/Users/steve/repos/PowerShell/src/System.Management.Automation/security/nativeMethods.cs(1205,17): error CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370. [/Users/steve/repos/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
Execution of { dotnet $Arguments } by build.psm1: line 374 failed with exit code 1
At /Users/steve/repos/PowerShell/build.psm1:1999 char:17
+                 throw $errorMessage
+                 ~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (Execution of { dotn\u2026ed with exit code 1:String) [], RuntimeException
+ FullyQualifiedErrorId : Execution of { dotnet $Arguments } by build.psm1: line 374 failed with exit code 1

```

Looks like it's due to use of `#pragma warning enable xxxx`.  Best I could find `latest` points to last released (non-Preview) which would be 7.3.  Seems like a bug in the compiler, opened https://github.com/dotnet/roslyn/issues/32958.  We can keep this workaround for now to make progress.